infra-as-code/bicep/modules/storagePrivateEndpoints.bicep

```diff
@@ -0,0 +1,75 @@
+@description('Name of the pe')
+param name string = ''
+
+@description('Location of the pe')
+param location string = ''
+
+@description('Id of the vnet.')
+param vnetId string = ''
+
+@description('Id of the subnet that the bastion will be deployed to.')
+param subnetId string = ''

```

Comment:
Instead of having both a subnet and a vnet being passed in, consider just passing in the subnetId and then calculating the vnetId from that.  Less chance of mis-use of the module by providing inconsistent resource Ids between the two params